exec/cnex/cell.c

```diff
@@ -336,6 +396,15 @@ Cell *cell_createDictionaryCell(void)
     return c;
 }
 
+Cell * cell_createPointerCell(void *p)
```

Comment:
Does this really need to be `void *`? Can it be an actual non-`void` pointer type?

Reply:
This was originally designed to allow the creation of "any" pointer type, maybe not even a Cell type.  I'm not even sure I need the pointer code, at this point.

Reply:
Well this function is not used anywhere here, so remove it.